src/Roslyn.Diagnostics.Analyzers/Core/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+
+namespace Roslyn.Diagnostics.Analyzers.Extensions
+{
+    internal static class INamedTypeSymbolExtensions
```

Comment:
Can these extension methods be added here: https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/Compiler/Extensions/INamedTypeSymbolExtensions.cs